tests/test_jointcal_cfht.py

```diff
@@ -282,34 +284,45 @@ def test_jointcalTask_2_visits_constrainedMagnitude_no_astrometry(self):
         # The resulting fit should be close to the constrainedFlux model:
         # there are few CCDs and 2 visits, so there's not a lot of complexity
         # in this case to distinguish the flux vs. magnitude models.
-        metrics['photometry_final_chi2'] = 3332.76
+        metrics['photometry_final_chi2'] = 9455.06
+        metrics['photometry_final_ndof'] = 2710
 
         self._testJointcalTask(2, None, None, pa1, metrics=metrics)
 
     def test_jointcalTask_2_visits_constrainedFlux_pedestal(self):
+        """Test that forcing a systematic flux error results in a lower chi2.
+        """
         pa1, metrics = self.setup_jointcalTask_2_visits_constrainedPhotometry()
-        self.config.photometryErrorPedestal = 0.02
+        # median fluxErr/flux in ps1 is 0.11, so we have to make this bigger
+        # than that to actually allow more slop in the fit.
+        self.config.photometryErrorPedestal = 0.2
 
         # We're allowing more error in the fit, so PA1 may be worse.
-        pa1 = 0.021
+        pa1 = 0.21
```

Comment:
This looks like a significant change, here.  Why is that?

Reply:
I had to increase the error pedestal a lot to get the final fit to look markedly different. Doing that means that's a lot more slop in the final fit.